components/DetailPullup.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { StyleSheet, Image, FlatList } from "react-native";
+import { View, Text } from "./Themed";
+
+const item = {
+  title: "Blue Pants",
+  user: "ravina patel",
+  description: `
+  Vintage blue pants that are super comfy and cool!
+  Condition
+  ISBN
+  Additional Information
+  `,
+  price: 25.00
+}
+
+const similarItems: string[] = [
```

Comment:
just a suggestion, you might want this to be image URLs instead of names so that you can display the images